docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,142 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
+
+**Category**: *Generation*
+
+**Short description**: *EyeLike* operation generates shift matrix.
+
+**Detailed description**:
+
+*EyeLike* operation generates a matrix with ones on the diagonal and zeros everywhere else. The index of the diagonal to be populated with ones is given by `diagonal_index`: `output[i, i + diagonal_index] = 1`.
+
+
+Example 1. *EyeLike* output with `output_type` = f32:
+
+``` 
+num_rows = 3
+
+output  = [[1. 0. 0.]
+           [0. 1. 0.]
+           [0. 0. 1.]]
+```
+
+Example 2. *EyeLike* output with `output_type` = i32, `diagonal_index` = 2:
+
+``` 
+num_rows = 3
+
+num_columns = 4
+
+output  = [[0 0 1 0]
+           [0 0 0 1]
+           [0 0 0 0]]
+```
+
+Example 3. *EyeLike* output with `output_type` = i32, `diagonal_index` = -1:
+
+``` 
+num_rows = 3
+
+num_columns = 4
+
+output  = [[0 0 0 0]
+           [1 0 0 0]
+           [0 1 0 0]]
+```
+
+Example 4. *EyeLike* output with `output_type` = f16, `diagonal_index` = 5:
+
+``` 
+num_rows = 2
+
+output  = [[0. 0.]
+           [0. 0.]]
+```
+
+**Attributes**:
+
+* *output_type*
+
+    * **Description**: the type of the output
+    * **Range of values**: any numeric type
+    * **Type**: string
+    * **Required**: *Yes*
+
+* *diagonal_index*
```

Comment:
let us make this as an input. It will be more flexible and it doesn't influence on output shape